ci-operator/step-registry/cucushift/cucushift-commands.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# prepare oc client
+mkdir /tmp/bin
+export PATH=$PATH:/tmp/bin
+curl https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz | tar xvzf - -C /tmp/bin/ oc
```

Comment:
Please inject the CLI using this doc: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli

So on the  `ci-operator/step-registry/cucushift/cucushift-ref.yaml` use `cli:latest` to get the correct `oc` for the version of OCP being tested